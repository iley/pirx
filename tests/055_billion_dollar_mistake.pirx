func main(): int {
  var x: int;
  var p: *int = null;
  printf("%p\n", p);

  if p == null {
    printf("null is null\n");
  }

  p = &x;
  if p == null {
    printf("this cannot be\n");
  }

  return 0;
}
