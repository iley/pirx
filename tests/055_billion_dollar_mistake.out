0
null is null
